Formula/qt5.rb

```diff
@@ -90,12 +100,32 @@ def install
       -qt-pcre
       -nomake tests
       -no-rpath
+      -pch
+      -system-proxies
+      -optimized-tools
```

Comment:
-pch enable precompiled-headers to speed up the compile process, with this, the compile process can speed up to 30%-40%, so why not use it? unless you prefer the slow compile.
-system-proxies, use system proxies setting instead of qt's own proxy setting in qtnetwork module, if you don't have this, anyone what to use the proxy in qt-network programs will need to set another proxy settings within qt itself, so I doubt anyone would do that.
-optimized-tools acutally, this does nothing here, it just make sure the qmake/moc/uic/rcc are build without debug info, in release build, this is the default setting.